pygerduty/events.py

```diff
@@ -0,0 +1,78 @@
+# Events module for pygerduty version 2. These methods are compatible with
+# Pagerduty Events API.
+
+import urllib
+from v2 import (
+    Pagerduty,
+    IntegrationAPIError,
+    INTEGRATION_API_URL,
+)
+
+class Events(Pagerduty):
```

Comment:
The trigger methods don't really benefit from having access to the rest api client or any shared state other than reusing `execute_request`. Maybe we can move the `execute_request` into a `common` module for reuse and make these trigger methods just be top-level functions.